src/chapters/data-manipulation/exercises.rst

```diff
@@ -1,2 +1,24 @@
 Exercises: Data Manipulation
-============================
\ No newline at end of file
+============================
+
+.. _dataManipulationNotebook:
+
+Getting Started
+---------------
+
+| The exercises have been combined with the reading for this chapter.  
+
+Fork this GitHub repository: `Prep Work Jupyter Notebooks <https://github.com/speudusa/DataManipulation>`__
```

Comment:
I don't think you need lines 11-13.  I feel that it is confusing and makes me doubt if i started the correct notebook work.

Reply:
Maybe even line 15 could go if you reference/link back to the reading in line 9?

Reply:
The Readings are just contained in the notebooks.  I'll add a note and see if that helps with confusion.

Reply:
Reworked both data-manipulation.rst and exercises.rst, thoughts?